test/sql/aggregate/aggregates/test_state_export.test

```diff
@@ -204,3 +204,35 @@ query IIIIIII nosort res10
 SELECT g, finalize(count_star_state),finalize(count_state), finalize(sum_state), finalize(avg_state)::integer, finalize(min_state), finalize(max_state) FROM state_view ORDER BY g;
 ----
 
+# BLOB casting back and forth
+statement ok
+SELECT (SUM(42) EXPORT_STATE)::BLOB;
+
+query I
+SELECT FINALIZE(COMBINE(SUM(42) EXPORT_STATE, (SUM(42) EXPORT_STATE)::BLOB));
+----
+84
+
+statement error
+SELECT (SUM(42) EXPORT_STATE)::INTEGER;
+
+
+# simulate round tripping
+require parquet
+
+statement ok
```

Comment:
@vincent-tr here is an example

Reply:
Please note that the `BLOB` needs to be the second argument to `COMBINE`

Reply:
Awesome!
thanks that you pointed it to me :)